files/gridlayout/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "gridlayout",
+  "repo": "ghinda/gridlayout",
+  "files": {
+    "include": ["gridlayout.css", "gridlayout.min.css", "gridlayout-ie.js", "gridlayout-ie.min.js"]
```

Comment:
Can you remove the unminified files from here? They are not required in production